### ISSUE TITLE
webhook create with filter condition - failure

### ISSUE THREAD

**Issue by dstala** [nocodb/nocodb#1657]
Node: **v14.18.0**
Arch: **arm64**
Platform: **darwin**
Docker: **false**
Database: **mysql2**
ProjectOnRootDB: **false**
RootDB: **sqlite3**
PackageVersion: **0.84.16**

- install discord webhook
- add webhook
- configure 'condition'
- save webhook

```
POST /api/v1/db/meta/tables/md_fzhjdclvyuzrw4/hooks 200 546 - 9.217 ms
projectAclMiddleware :: Forbidden [Error]: filterCreate - user,super : Not allowed
    at Function.forbidden (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/noco/meta/helpers/catchError.ts:47:11)
    at projectAclMiddleware (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/noco/meta/helpers/ncMetaAclMw.ts:63:17)
    at /Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/noco/meta/helpers/catchError.ts:6:20
    at Layer.handle [as handle_request] (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/node_modules/express/lib/router/route.js:137:13)
    at authMiddleware (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/noco/meta/helpers/ncMetaAclMw.ts:23:7)
    at /Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/noco/meta/helpers/catchError.ts:6:20
    at Layer.handle [as handle_request] (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/node_modules/express/lib/router/route.js:137:13)
    at exports.default (/Users/rajuudava/NocoDB/nocodb/packages/nocodb/src/lib/noco/meta/helpers/extractProjectIdAndAuthenticate.ts:141:5)
POST /api/v1/db/meta/views/hk_iupzihwaar9949/filters 403 49 - 2.581 ms
GET /api/v1/db/meta/tables/md_fzhjdclvyuzrw4/hooks 200 557 - 1.464 ms
GET /api/v1/db/meta/hooks/hk_iupzihwaar9949/filters 200 2 - 2.473 ms

```

**Comment by wingkwong** [comment]
Couldn't reproduce. it seems that this issue has been fixed in https://github.com/nocodb/nocodb/commit/adc6045c7a3b83bd2545d76e01ea3b824cc2c362. @dstala mind to retest it with the latest code?

**Comment by wingkwong** [comment]
Fixed in ``v0.90.7``.
